README.md

```diff
@@ -9,6 +9,23 @@ tuple and performs the expiry itself: either deletes it (memcache), or
 does something smarter, like put a smaller representation of the data
 being deleted into some other space.
 
+There are a number of similar modules:
+- [moonwalker](https://github.com/tarantool/moonwalker) triggered manually,
+useful for batch transactions, a performance about 600/700k rec/sec
+- [expirationd](https://github.com/tarantool/expirationd/issues/53) always
+expires tuples without using indices but using any condition, without guarantee
+for time expiration.
```

Comment:
> always expires tuples without using indices

Now it is not quite correct, right? See #50 and PR #52.

Reply:
Probably yes. Although proposed table is good enough to make an opinion about these projects.